src/constants.ts

```diff
@@ -138,6 +138,7 @@ export enum RENDERING_MODES {
 export const GLOBAL_SLIDER_DIMENSION_FIELDS = ['z', 't'] as const;
 
 export enum INTERPOLATION_MODES {
+  AUTO,
```

Comment:
Got a "Computed values are not permitted in an enum with string valued members." error with this set as a string so this seemed like a good way around that